src/components/Settings.vue

```diff
@@ -3,15 +3,15 @@
     <h3 :class="{'text-white': isDarkMode}">Settings</h3>
     <b-form-checkbox @change="setCbMode" :class="[{'text-white': isDarkMode}]" v-model="isCbMode" name="cbModeSwitch"
                      v-b-tooltip.hover.lefttop :title="cbExplanation" switch>
-      Colorblind Mode
+      Color-Blind Mode
     </b-form-checkbox>
     <b-form-checkbox @change="setDarkMode" :class="[{'text-white': isDarkMode}]" v-model="isDark" name="DarkModeSwitch"
                      v-b-tooltip.hover.lefttop :title="darkExplanation" switch>
       Dark Mode
     </b-form-checkbox>
     <b-form-checkbox  @change="setAdvMode" :class="[{'text-white': isDarkMode}]" v-model="isAdvMode" name="AdvModeSwitch"
                       v-b-tooltip.hover.lefttop :title="advSettingsExplanation" switch>
-      Enable Advanced Settings
+      {{isAdvMode?'Hide':'Enable'}} Advanced Settings

```

Comment:
The idea behind having the title of the advanced settings toggle switch between “Hide” and “Show” was that it would make more sense in the context of a button. It looks like you implemented the changing title without switching from a toggle switch to a regular button.

Here’s [the documentation](https://getbootstrap.com/docs/5.2/components/buttons/#toggle-states) for a Bootstrap component that resembles what I’m envisioning. Notice that when you click on the button the first time, it becomes darker to indicate that it’s in the _enabled_ state. When you click on it a second time, it goes back to its original appearance, indicating a return to the _disabled_ state.

In our case, the title of the button in the _disabled_ (default) state should be “Show Advanced Settings“, and the title in the _enabled_ state should be “Hide Advanced Settings“.

Reply:
I changed it to a regular button
![image](https://user-images.githubusercontent.com/80116509/192631322-de2cc97b-8e30-4ab8-8c07-a8413b52b8cb.png)
![image](https://user-images.githubusercontent.com/80116509/192631342-9f130d88-9033-4dc2-883b-79de5ed81547.png)